metrics/record_test.go

```diff
@@ -186,3 +188,43 @@ func TestBucketsNBy10(t *testing.T) {
 		})
 	}
 }
+
+func TestMeter(t *testing.T) {
+	measure := stats.Int64("request_count", "Number of reconcile operations", stats.UnitNone)
+	// Increase the measurement value for each test case so that checking
+	// the last value ensures the measurement has been recorded.
+	meterTestCases := []cases{{
+		name:          "resource 1",
+		metricsConfig: &metricsConfig{},
+		measurement:   measure.M(1),
+		resource:      resource.Resource{Type: "resource 1", Labels: map[string]string{"bar": "foo"}},
+	}, {
+		name:          "resource 2",
+		metricsConfig: &metricsConfig{},
+		measurement:   measure.M(2),
+		resource:      resource.Resource{Type: "resource 2", Labels: map[string]string{"bar": "foo", "bar1": "foo1"}},
+	}}
+	testMeterRecord(t, measure, meterTestCases)
+}
+
+func testMeterRecord(t *testing.T, measure *stats.Int64Measure, tests []cases) {
```

Comment:
FWIW, I don't see why do we need this helper method at all?

Reply:
It was copied from a pattern above, but happy to remove it.